Core/TokenServices/TokenGenerationServices/ITokenGenerationService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Data.Models;
+
+namespace Core.TokenServices.TokenGenerationService
+{
+    public interface ITokenGenerationService
+    {
+        List<Token> GenerateTokens(int numberOfTokens);
```

Comment:
The Return type should be of type IEnumerable and not List. IEnumerable is to be used because we intend not to modify the data returned, hence it is of  "read only" type. That is why we need to use IEnumerable.